cumulus/pallets/xcmp-queue/src/benchmarking.rs

```diff
@@ -85,7 +85,7 @@ mod benchmarks {
 		}
 
 		assert!(
-			OutboundXcmpStatus::<T>::get().iter().find(|p| p.recipient == para).is_none(),
+			!OutboundXcmpStatus::<T>::get().iter().any(|p| p.recipient == para),
```

Comment:
slightly cleaner maybe?
```suggestion
			OutboundXcmpStatus::<T>::get().iter().all(|p| p.recipient != para),
```

Reply:
Nice, agreed. I wonder which one De Morgan would prefer